messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/JGroupBroadcastGroupRemove.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2011, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.extension.messaging.activemq;
+
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OP_ADDR;
+
+import org.jboss.as.controller.AbstractRemoveStepHandler;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.capability.DynamicNameMappers;
+import org.jboss.as.controller.capability.RuntimeCapability;
+import org.jboss.dmr.ModelNode;
+
+/**
+ * Removes a broadcast group.
+ *
+ * @author Brian Stansberry (c) 2011 Red Hat Inc.
+ */
+public class JGroupBroadcastGroupRemove extends AbstractRemoveStepHandler {
+    public static final RuntimeCapability<Void> CAPABILITY = RuntimeCapability.Builder.of("org.wildfly.messaging.activemq.broadcast-group", true)
+            .setDynamicNameMapper(DynamicNameMappers.PARENT)
+            .build();
+
+    public static final JGroupBroadcastGroupRemove INSTANCE = new JGroupBroadcastGroupRemove(true);
+    public static final JGroupBroadcastGroupRemove LEGACY_INSTANCE = new JGroupBroadcastGroupRemove(false);
```

Comment:
Same question about legacy vs non-legacy behavior in the same handler.

Reply:
legacy means that the handler has been called from a legacy operation command call